src/EditStudent.java

```diff
@@ -0,0 +1,59 @@
+import java.util.*;
+
+public class EditStudent implements ManagementActionFunction {
+    // Reset color
+    public static final String ANSI_RESET = "\u001B[0m";
+    // Red color
+    public static final String ANSI_RED = "\u001B[31m";
+
+    @Override
+    public void action(ManagementApplication managementApplication) {
+        Scanner scanner = new Scanner(System.in);
+        Map<Integer, Object> studentMap = managementApplication.getStudentMap();
+
+        // 학생 ID 입력 받기
+        System.out.print("수정할 수강생의 ID를 입력해주세요 : ");
+        int idToChange = scanner.nextInt();
+        scanner.nextLine();
+
+        Student student = (Student) studentMap.get(idToChange);
+        if (student != null) {
```

Comment:
```suggestion
        if (student == null) {
            System.out.println(ANSI_RED + "ID가 " + idToChange + "인 수강생의 정보가 없습니다." + ANSI_RESET);
            return;
        }
```

위처럼 찾은 학생이 없을때를 먼저 처리해 들여쓰기를 한칸 줄일 수 있을거 같습니다!